docs/data/hadoop-overview.md

```diff
@@ -44,24 +56,38 @@ Minimal Installation (0-50TB, WAN transfers up to 1Gbps)
+- hadoop-name: The namenode for the Hadoop system.  Must be on a private network.
+- hadoop-name2: This will run the HDFS secondary namenode. Must be on a private network.
+- hadoop-data1, hadoop-data2: Two HDFS datanodes. They will hold data for the system, so they should have sizable hard drives. As the Hadoop installation grows to many terabytes, this will be the only class of nodes one adds. Must be on a private network.
+- hadoop-grid: Runs the Globus GridFTP server. Must have a public interface and a private interface.
 
-If desired, hadoop-name and hadoop-name2 may be virtualized. Prior to installation, DNS / host name resolution **must** work. That is, you should be able to resolve all the hadoop servers either through DNS or /etc/hosts. Because of the grid software, hadoop-grid **must** have reverse DNS working.
+If desired, hadoop-name and hadoop-name2 may be virtualized. 
+Prior to installation, DNS / host name resolution **must** work. 
+That is, you should be able to resolve all the hadoop servers either through DNS or /etc/hosts. 
+Because of the grid software, hadoop-grid **must** have reverse DNS working.
 
 Medium Installation (50-150TB, WAN transfers up to 2 Gbps)
 ----------------------------------------------------------
 
-For a medium install, make the following changes over the minimal install: 1 Run BeStMan on a separate machine, hadoop-srm. This host may be virtualized. 1 Run multiple GridFTP servers (2 should be fine); if possible, use 10 Gbps cards for these hosts. 1 Add many more HDFS datanodes. This usually means starting to add 1 or 2 TB hard drives to some worker nodes.
+For a medium install, make the following changes over the minimal install: 
+
+- Run multiple GridFTP servers (2 should be fine); if possible, use 10 Gbps cards for these hosts. 
+- Add many more HDFS datanodes. This usually means starting to add 1 or 2 TB hard drives to some worker nodes.
 
 Large Installation (>150TB, WAN transfers over 2 Gbps)
 ---------------------------------------------------------
 
-For a large installation, make the following changes: 1 Run more GridFTP servers; plan for 800 Mbps per host with 1 Gbps card in order to have excess capacity. 1 Purchase machines suitable for HDFS datanodes. It is possible to get a 2U box with 8-12 hard drives; for many projects, this will provide a more suitable CPU to disk ratio than the 1U boxes with 2 hard drives.
+For a large installation, make the following changes: 
+
+- Run more GridFTP servers; plan for 800 Mbps per host with 1 Gbps card in order to have excess capacity. 
+- Purchase machines suitable for HDFS datanodes. It is possible to get a 2U box with 8-12 hard drives; for many projects, this will provide a more suitable CPU to disk ratio than the 1U boxes with 2 hard drives.
```

Comment:
Are the small/medium/large installation numbers still correct? What if the site decides to run xrootd-hdfs instead of gridftp?

Reply:
Checking with Garhan at UNL about the numbers, don't know if the values are still correct.

Reply:
Garhan said the numbers are still appropriate, except for UNL dedicates 1.5GB on HDFS datanodes.  Updated the text to say 1-1.5GB for datanodes since UNL has a lot of IO heavy workflows that use a lot of RAM on the data nodes.  

Also, Garhan suggested doing away with the small,medium, large installation bits since the cluster scales horizontally.  I agree with this and changed the text to just indicate that admins just need to add datanodes and/or gridftp transfer nodes as cluster size increases.